### ISSUE TITLE
Add last aggregator to datadog_timeboard resource

### ISSUE THREAD

**Issue by jesusvazquez** [hashicorp/terraform#14390]
### Terraform Version
```
$ terraform -v                          
Terraform v0.9.4
```
### Affected Resource(s)
[datadog_timeboard](https://www.terraform.io/docs/providers/datadog/r/timeboard.html)
### Terraform Configuration Files
```hcl
  graph = {
    title = "Consul Current Latency"
    viz = "query_value"

    request = {
        q = "avg:consul.net.node.latency.p95{$env,$consul_datacenter}"
        aggregator = "last" <--
        type = "line"
      }
      autoscale = true
      custom_unit = "ms"
  }

```

### Expected Behavior
I expect terraform to recognize the aggregator last but it fails (see next point). 

### Actual Behavior
``` 
1 error(s) occurred:

* datadog_timeboard.consul: "graph.3.request.0.aggregator" contains an invalid method "last". Valid methods are either "average", "max", "min", or "sum"
```

### Steps to Fix
Modify [this](https://github.com/hashicorp/terraform/blob/master/builtin/providers/datadog/resource_datadog_timeboard.go#L704-L717)

### Important Factoids
Datadog does have the **last** aggregator available:
![image](https://cloud.githubusercontent.com/assets/2439858/25947419/a2ec471a-364f-11e7-8d99-ec499d52371c.png)

The datadog api returns this value:
```
      {
        "definition": {
          "viz": "query_value",
          "requests": [
            {
              "q": "avg:consul.net.node.latency.p95{$env,$consul_datacenter}",
              "aggregator": "last",
              "conditional_formats": [],
              "type": "line"
            }
          ],
          "autoscale": true,
          "custom_unit": "ms"
        },
        "title": "Consul Current Latency"
      },
``` 

### References
None

**Comment by jesusvazquez** [comment]
I believe #14391 fix this issue.

**Comment by stack72** [comment]
Closed via #14391
